test-count.ps1

```diff
@@ -0,0 +1,58 @@
+[CmdletBinding()]
+param (
+  [string]$GitRepoPath = '.',
+  [string]$E2ePath = 'Tests/E2E',
+  [string]$IntegrationPath = 'Tests/Integration' ,
+  [string]$UnitPath = 'Tests/Unit',
+  [string]$TestAttributeRegex = '\[ *(Fact|Theory) *\]',
+  [string]$OutputFile = 'test-count.csv'
+)
+
+$ErrorActionPreference = 'Stop'
+
+$fullOutputPath = Join-Path -Path $PWD -ChildPath $outputFile
+
+Write-Host "Current directory: " $PWD " OutputPath: " $fullOutputPath
+Clear-Content -Path $fullOutputPath -ErrorAction SilentlyContinue
+Push-Location $gitRepoPath
+Write-Host "Changed to repository directory: " $PWD
+
+Out-TestCountCsv 'Integration' $integrationPath $fullOutputPath
+Out-TestCountCsv 'E2E' $e2ePath $fullOutputPath
+Out-TestCountCsv 'Unit' $unitPath $fullOutputPath
+
+Pop-Location
+
+Write-Host "Done, Curent Directory: " + $PWD
```

Comment:
[nit] It's still showing 'Curent' for me, could you check again @roel4ez ?

Reply:
sorry didn't push the changes yet, fixed now